client/src/app/app.component.scss

```diff
@@ -0,0 +1,22 @@
+.subscribe-banner {
+    color: #fff;
+    position: absolute;
+    bottom: 20px;
+    width: 75%;
+    left: 12.5%;
+    text-align: center;
+  }
+  
+  .silver-modal {
```

Comment:
I'd remove this scss if it's not being used.

Reply:
Whoops thanks, had copied it then removed the usage because the silver store is probably a much later priority